package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "User-api",
+  "version": "1.0.0",
+  "description": "A simple CRUD API supported by a postgres database",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
Wouldn't `exit 1` here, unless you're going to think about adding tests later on.